.github/workflows/elixir.yml

```
@@ -7,52 +7,59 @@ name: Elixir CI
 
 on:
   push:
-    branches: [ "master" ]
+    branches: ["master"]
   pull_request:
-    branches: [ "master" ]
+    branches: ["master"]
 
 permissions:
   contents: read
 
 jobs:
   build:
-
     name: Build and test
     runs-on: ubuntu-latest
-
+    strategy:
+      matrix:
+        elixir: ["1.15.8", "1.16.3", "1.17.3", "1.18.2"]
+        otp: ["25.3.2", "26.2.5", "27.2.4"]
+        exclude:
+          - elixir: "1.15.8"
+            otp: "27.2.4"
+          - elixir: "1.16.3"
+            otp: "27.2.4"
```

```suggestion
        include:
          - elixir: "1.15"
            otp: "25"
          - elixir: "1.18"
            otp: "27"
            lint: true
```

I don't think we need a _whole_ matrix with intermediate Elixir/OTP versions. It's simpler to just cover a very old Elixir version and the most recent stable.